mapeditor/resourceExtractor/ResourceConverter.h

```diff
@@ -24,26 +24,34 @@ struct ExtractionOptions
 {
 	bool extractArchives = false;	// if set, original H3 archives will be extracted into a separate folder
 	ConversionOptions conversionOptions;
+	bool moveExtractedArchivesToSoDMod = false;
 };
 
+/**
+* Class functionality to be used after extracting original H3 resources and before moving those resources to SoD Mod
+* Splits def files containing individual images, so that faction resources are independent. (TwCrPort, CPRSMALL, FlagPort, ITPA, ITPt, Un32 and Un44)
+* (town creature portraits, hero army creature portraits, adventure map dwellings, small town icons, big town icons,
+* hero speciality small icons, hero speciality large icons)
+* Converts all PCX images to PNG
+ */
 class ResourceConverter
 {
 
 public:
 
-	// Splits def files that are shared between factions and converts pcx to bmp depending on Extraction Options
+	// Splits def files that are shared between factions and converts pcx to PNG depending on Extraction Options
 	static void convertExtractedResourceFiles(ConversionOptions conversionOptions);
 
 private:
 
-	// converts all pcx files from /Images into PNG
-	static void doConvertPcxToPng(bool deleteOriginals);
+	/** Converts all .pcx from extractedFolder/Images into .png */
```

Comment:
We generally use single-line comments, e.g. `//` or triple slash `///` for documentation.
I think only file headers use multi-line comments for some reason

Reply:
got it. Using /// and // for comments